posydon/binary_evol/CE/step_CEE.py

```diff
@@ -484,6 +492,10 @@ def CEE_simple_alpha_prescription(
         core_definition_He_fraction: float
             The value of the He abundance to define the envelope-core boundary
             in CO_cores.
+        mass_loss_during_CEE_merged: Boolean
+             If False, then no mass loss from this step for a merged CEE
+             If True, then we remove mass according to the alpha-lambda prescription
+              assuming a final separation where the inner core(s) RLOF starts, and the same lambda(s)
```

Comment:
Using the same `lambda` is a bad idea. Can't we use the profiles?

Reply:
Now the binding energy is calculated through the profile, not really caring about a lambda for the partial mass ejection